### PR TITLE
Fixed mobile view in admin tab

### LINKED ISSUE #2357: Fix mobile view in admin page
**Actual Behaviour**

Mobile view is not proper in the admin page

**Expected Behaviour**

Fix mobile view in the admin page

**Screenshots of the issue**

Capture

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2357

Changes: Fixed mobile view in the admin page

Demo Link: https://pr-2401-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
addddd

### CODE REVIEW

**Comment by samagragupta** [comment]
@akshatnitd Made the changes

**Comment by harsh-jindal** [comment]
@samagragupta Add margin to the left of User Roles and skill types Card
Screenshot 2019-07-02 at 9 59 55 AM

**Comment by samagragupta** [comment]
@harsh-jindal @fragm3 Made the changes
Capture

**Comment by mariobehling** [comment]
> @harsh-jindal @fragm3 Made the changes
> Capture

What screen resolution are you using? Please try to fit three cards into one row.

**Comment by samagragupta** [comment]
@mariobehling @akshatnitd Made the requested changes

**Comment by samagragupta** [comment]
@akshatnitd 
Capture

**Comment by samagragupta** [comment]
@fragm3 Made the changes

**Comment by mariobehling** [comment]
Looks good for me. We also need to test this a bit and we will add more analytics to this area. So, it will change a lot anyways. Let's go step by step. Thank you!

File: src/components/Admin/AdminTab/AdminTab.js [hunk 1]
```diff
@@ -18,16 +18,58 @@ const CardContentContainer = styled.div`
 const CardContainer = styled.span`
   font-size: 18px;
   float: left;
-  margin-right: 20px;
+  margin-right: 10px;
```
Comment (fragm3) [inline src/components/Admin/AdminTab/AdminTab.js]: margin: 0 10px;

File: src/components/Admin/AdminTab/AdminTab.js [hunk 2]
```diff
@@ -18,16 +18,58 @@ const CardContentContainer = styled.div`
...
+  margin-left: 10px;
+  @media (max-width: 894px) {
+    margin-left: 0px;
```
Comment (fragm3) [inline src/components/Admin/AdminTab/AdminTab.js]: margin: 0px;

File: src/components/Admin/AdminTab/AdminTab.js [hunk 3]
```diff
@@ -18,16 +18,58 @@ const CardContentContainer = styled.div`
...
+    font-size: 1rem;
+  }
+`;
+
+const Container = styled.div`
+  width: 840px;
+  margin: auto;
+  @media (min-width: 1319px) {
+    ${CardContainer}:nth-child (4) {
+      display: table;
```
Comment (fragm3) [inline src/components/Admin/AdminTab/AdminTab.js]: This could be done other way as well, look into this

**Comment by samagragupta** [comment]
@fragm3  Made the changes

**Pull request merged**